.circleci/config.yml

```diff
@@ -197,11 +374,17 @@ releasable_branches: &releasable_branches
       - main
 
 workflows:
+  # Tells CircleCI to skip this workflow when the pipeline is triggered by the scheduled source,
```

Comment:
*nit:* should this comment be below `test_deploy`, instead of above it? Seems like it is relevant for `when`.